### ISSUE TITLE
Will training logs be shared?

### ISSUE THREAD

**Issue by hetong007** [Microsoft/human-pose-estimation.pytorch#74]
Hi, 

I'm re-implementing your algorithm, and would love to compare the training logs for consistency. 

So is there any plan to share the training logs?

Thanks!

**Comment by hengck23** [comment]
you can run the author code and you will get the same results.

i also re-implement the code and here are my training log (loss curve) and results (mAP 0.709 for resnet50_192x256). Note that i only train for 130 epoch (instead of 140). flip-test is disabled and i am using coco ground truth box as input. the way i compute the pck_h accuracy is a little different. please see the links:

https://gist.github.com/hengck23/9b0930a07110f801da8ec49cb2800def

https://gist.github.com/hengck23/0d15a792c5ddd99dab94a8997ef0b493

**Comment by hetong007** [comment]
@hengck23 That's very helpful. Much appreciated!

**Comment by edwardnguyen1705** [comment]
Hi @hengck23 could you tell me why `0.1` here `rx, ry = [0.1*W, 0.1*H] #[0.1*H, 0.1*W]   #` 
Thanks!
